src/SprykerSdk/Integrator/IntegratorConfig.php

```diff
@@ -24,14 +22,37 @@ class IntegratorConfig extends AbstractBundleConfig
     public const MANIFEST_KEY_POSITION_BEFORE = 'before';
     public const MANIFEST_KEY_POSITION_AFTER = 'after';
 
+    public const CORE_NAMESPACES = 'CORE_NAMESPACES';
+    public const PROJECT_NAMESPACES = 'PROJECT_NAMESPACES';
+
+    /**
+     * @var array|null
+     */
+    protected $config;
+
+    /**
+     * @return void
+     */
+    public function loadConfig(): void
+    {
+        if ($this->config === null) {
+            $fileName = $this->getConfigPath();
+            if (file_exists($fileName)) {
+                include $fileName;
```

Comment:
The idea was to merge default Spryker config with the dedicated integrator config. But currently we just get one of them. Please fix this